### DISCUSSION TITLE
Advanced/Debug

### DISCUSSION THREAD

**Discussion by giscus** [py32duino/document#5] [comment]
# Advanced/Debug

使用 VSCode 的Crotex-Debug插件 先决条件 安装 VSCode; 有一个支持的调试器，可以是 JLink，ST-Link，CMSIS-DAP 等，这里我们使用 DAPLink 作为例子; 安装 PY32Duino 的 SDK; 安装PyOCD; VSCode 的配置 安装插件 在 VSCode 的插件市场中搜索Cortex-Debug...

https://arduino.py32.org/Advanced/Debug.html

**Comment by changlin1990** [comment]
请问用arduino IED自带的调试功能如何调试？

**Comment by changlin1990** [comment]
你好，
开发板管理区地址好像有问题，打不开，上面一个地址





发自我的iPhone


------------------ Original ------------------
From: giscus[bot] ***@***.***&gt;
Date: Fri,Mar 1,2024 4:41 PM
To: py32duino/document ***@***.***&gt;
Cc: changlin1990 ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [py32duino/document] Advanced/Debug (Discussion #5)





 
Advanced/Debug
 
使用 VSCode 的Crotex-Debug插件 先决条件 安装 VSCode; 有一个支持的调试器，可以是 JLink，ST-Link，CMSIS-DAP 等，这里我们使用 DAPLink 作为例子; 安装 PY32Duino 的 SDK; 安装PyOCD; VSCode 的配置 安装插件 在 VSCode 的插件市场中搜索Cortex-Debug...
 
https://arduino.py32.org/Advanced/Debug.html

**Comment by be-engineer** [comment]
py32f003x8编译报错
